bHY6yh-de9615/cod-vyg67d_bHY6yh-de9615.xml

```diff
@@ -0,0 +1,120 @@
+           <lb ed="#O"/> ¶ Cenantibus iesu et discipulis 
+           <lb ed="#O"/>eius accepit iesus panem et 
+           <lb ed="#O"/>benedixit ac fregit deditque discipu<lb ed="#O" break="no"/>lis 
+           suis et ait illis ¶ Accipite et co<lb ed="#O" break="no"/>medite
+           hoc est corpus meum et Ac<lb ed="#O" break="no"/>cipiens 
+           calicem grans egit et dedit 
+           <lb ed="#O"/>illis dicens bibite ex hoc omnes hic 
+           <lb ed="#O"/>est enim sanguis meus noui testa<lb ed="#O" break="no"/>menti 
+           qui pro multis effundetur in
+           <lb ed="#O"/>remissionem peccatorum.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9615-d1e148">
+           <!--line bumps down-->
+           <lb ed="#O"/> ¶ <ref xml:id="bHY6yh-de9615-Rd1e148">Luchas in euangelis capitulo 22</ref> <!--line bumps up--> <quote xml:id="bHY6yh-de9615-Qd1e152">Iesus in ce<!--line bumps back down-->na quam 
+           <lb ed="#O"/>agebat accepto panem gracias 
+           <lb ed="#O"/>egit et fregit et dedit eis dicens hoc
+           <lb ed="#O"/>est corpus meum quod pro uobis trada<lb ed="#O" break="no"/>tur. 
+           hoc facite in meam commemora<lb ed="#O" break="no"/>tionem. 
+           Similter et calicem postquam cena<lb ed="#O" break="no"/>uit 
+           dicens hic est calix nouum testa
+           <lb ed="#O"/>metum in sanguine meo qui pro uobis 
+           <lb ed="#O"/>effundetur</quote>
+        </p>
+        <p xml:id="bHY6yh-de9615-d1e170">
+           <!--line bumps down-->
+           <lb ed="#O"/><ref xml:id="bHY6yh-de9615-Rd1e172">Iohannes in <!--line bumps up--> euangelio capitulo 6.</ref> ¶ <quote xml:id="bHY6yh-de9615-Qd1e174">dixit <!--line bumps back down--> iesus Ego sum panis ui<lb ed="#O" break="no"/>te
+           <pb ed="#O" n="28-r"/>
+           <cb ed="#O" n="a"/>
+           qui uenit ad me non exuriet et 
+           <lb ed="#O"/>qui credit in me non sitiet unquam. 
+           <lb ed="#O"/> ¶ Ait iesus. Ego sum panis uiuus qui 
+           <lb ed="#O"/>de celo descendi siquis manducauerit 
+           <lb ed="#O"/>ex hoc pane uiuet in eternum. Et pa<lb ed="#O" break="no"/>nis 
+           quem ego dabo caro mea est pro 
+           <lb ed="#O"/>mundi uita. ¶ Dixit iesus nisi man<lb ed="#O" break="no"/>ducaueritis 
+           carnem filii hominis et bibe<lb ed="#O" break="no"/>ritis 
+           eius sanguinem non habebitis ui<lb ed="#O" break="no"/>tam 
+           in nobis. Qui manducat <unclear>et</unclear>
+           <lb ed="#O"/>meam carnem et bibit meum sanguinem 
+           <!--several senteces omitted-->
+           <lb ed="#O"/>in me manet et ego in illo.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9615-d1e206">
+           <lb ed="#O"/><ref xml:id="bHY6yh-de9615-Rd1e210">Paulus apostolus epistola. i. ad corinth. capitulo 11.</ref> <quote xml:id="bHY6yh-de9615-Qd1e210">
+           <lb ed="#O"/>Quicumquam manducauerit panem 
```

Comment:
One small requested change about where the quotation begins. In this case the quotation begins after the line break.

```suggestion
           <lb ed="#O"/><ref xml:id="bHY6yh-de9615-Rd1e210">Paulus apostolus epistola. i. ad corinth. capitulo 11.</ref> 
           <lb ed="#O"/><quote xml:id="bHY6yh-de9615-Qd1e210">Quicumquam manducauerit panem 
```